config.c

```diff
@@ -353,35 +353,41 @@ struct mddev_dev *load_partitions(void)
 struct mddev_dev *load_containers(void)
 {
 	struct mdstat_ent *mdstat = mdstat_read(0, 0);
+	struct mddev_dev *dev_list = NULL;
+	struct map_ent *map_list = NULL;
 	struct mdstat_ent *ent;
-	struct mddev_dev *d;
-	struct mddev_dev *rv = NULL;
-	struct map_ent *map = NULL, *me;
 
 	if (!mdstat)
 		return NULL;
```

Comment:
this check seems redundant, in case of mdstat == NULL, NULL will be returned through dev_list
`if (!mdstat)
	return NULL;
`

Reply:
Right, fixed.